### ISSUE TITLE
Missing War information...

### ISSUE THREAD

**Issue by braoult** [1n9i9c7om/ClashOfClans-API-PHP#21]
The following are not handled:

`get /clans/{clanTag}/warlog` : Clan war log
`get /clans/{clanTag}/currentwar` : Information about clan's current clan war
`get /clans/{clanTag}/currentwar/leaguegroup` :  Information about clan's current clan war league group
`get /clanwarleagues/wars/{warTag}` : Information about a clan war league war

**Comment by buster95** [comment]
your clan needs to have the war log in public in the clan settings to show that info.

hey dude, I'm development a gateway for COC-API in this repo https://github.com/buster95/clashofclans_gateway if you are interesting

**Comment by braoult** [comment]
> your clan needs to have the war log in public in the clan settings to show that info.

This is not a permission issue (I used my own clan). Did you even try with this API ?

**Comment by buster95** [comment]
I had a issue with my clan to get info about wars, war log needs to be in public domain

Sure, now I'm making a war bot for my clan in telegram
and building this webapp https://clashamigos.web.app

**Comment by braoult** [comment]
> I had a issue with my clan to get info about wars, war log needs to be in public domain

Everyone who used Clash API knows about war permissions, this is not the case in this issue, I don't understand your answer at all...

**Comment by buster95** [comment]
![image](https://user-images.githubusercontent.com/15637669/94886149-6c302280-042f-11eb-84d5-6a048cb1a1e2.png)

I was just trying to tell you that if you want to change web service you could use the one I am developing
